test/ModuleInterface/if-configs.swift

```diff
@@ -126,3 +126,28 @@ public func hasIfCompilerCheck(_ x: () -> Bool = {
 #endif
   }) {
 }
+
+// CHECK: func hasComments
+// CHECK-NOT: comment!
+public func hasComments(_ x: () -> Bool = {
+  /* comment! */ // comment!
```

Comment:
Stripping comments could make valid Swift code invalid. For example

```swift
let x =/*comment*/1
```

compiles after removing the comment, it doesn’t

```swift
let x =1 // error: '=' must have consistent whitespace on both sides
```

Reply:
Oof, I handled that in a previous revision, but lost it with the current technique. I have to append a whitespace character for comments (and possibly a newline for line comments)

Reply:
Yes, that’s the lesson that trivia aren’t so trivial after all 😉

Reply:
Addressed in the latest push, appending an unconditional whitespace character for comments